apps/state/test/state_test.exs

```diff
@@ -65,6 +65,31 @@ defmodule StateTest do
              ) == [three, one, two]
     end
 
+    test "sorting by distance returns error when missing lat/lng" do
+      items = [
+        one = %{latitude: 1.0, longitude: 2.0},
+        two = %{latitude: 1.0, longitude: 1.0},
+        three = %{latitude: 1.0, longitude: 3.0}
```

Comment:
This probably won't validate because you aren't using `one` `two` or `three`. You also probably only one or two items.